### ISSUE TITLE
pytest django40 warning: django.utils.translation.ugettext_lazy() is deprecated

### ISSUE THREAD

**Issue by Grreg** [jmfederico/django-use-email-as-username#8]
* Django use Email as Username version: django-use-email-as-username 1.1.3.dev0
* Django version: Django 3.1.5
* Python version: Python 3.9.1
* Operating System: Win10

### Description
pytest throws a waning that the use of  "**django.utils.translation.ugettext_lazy() is deprecated**"
```
============================================================================ warnings summary ============================================================================
..\..\..\users\greg\appdata\local\programs\python\python39\lib\site-packages\django_use_email_as_username\models.py:47
  c:\users\greg\appdata\local\programs\python\python39\lib\site-packages\django_use_email_as_username\models.py:47: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    email = models.EmailField(_("email address"), unique=True)
```

This warning is also thrown for:
* django_use_email_as_username\models.py:47
* django_use_email_as_username\admin.py:25
* django_use_email_as_username\admin.py:14
* django_use_email_as_username\admin.py:12

### What I did

Install pytest for my project, made basic configuration for pytest and ran it without any further personally configured tests.
```pip install pytest-django
pytest
```

I know this is not a critical thing, but maybe its interesting for you.

Thank you very much for your plugin! Helped a lot!

Best regards,
Greg

**Comment by jmfederico** [comment]
Hi @Grreg 

Thanks for the report, will include a fix in the next release.

**Comment by Grreg** [comment]
Wow, that's a crazy fast reply! Thanks a lot! I'm pretty new to Django, so I didn't know if I can simply replace it. Hope to offer you a pull-request next time :)

**Comment by jmfederico** [comment]
I think it is a matter of replacing with `gettext_lazy`. Under the hood it is just an alias (and has been one since Django 2 I think) so should not break anything.

The PR would be appreciated!

**Comment by jmfederico** [comment]
Also, I am glad the module helped 🙂.

**Comment by Grreg** [comment]
I tried my best, but I have no clue how to link this Pull request with this issue :D
https://github.com/jmfederico/django-use-email-as-username/pull/9

**Comment by jmfederico** [comment]
Fixed by #9
